Bottom: 0d299d4377e967f8503d747a129dfe55b9dedc71
Top:    141529ed2a37cc6bcca798d16726424fbad9f6fc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 16:26:06 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index e090e11..3e08fec 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -807,12 +807,6 @@ hidgl_reset_stencil_usage (void)
   dirty_bits = 0;
 }
 
-void
-hidgl_set_depth (float depth)
-{
-  global_depth = depth;
-}
-
 
 /* ---------------------------------------------------------------------- */
